django/test/runner.py

```diff
@@ -644,7 +664,9 @@ def reorder_suite(suite, classes, reverse=False):
     reordered_suite = suite_class()
     for i in range(class_count + 1):
         reordered_suite.addTests(bins[i])
-    return reordered_suite
+
+    reordered_suite_postprocess = reorder_postprocess(reordered_suite)
+    return reordered_suite_postprocess
```

Comment:
Drop the blank line, avoid the unnecessary temporary variable for the return statement.